gubernator/view_logs.py

```diff
@@ -18,7 +18,7 @@
 import log_parser
 import kubelet_parser
 import regex
-from view_base import RenderingHandler, memcache_memoize, gcs_ls
+from view_base import BaseHandler, memcache_memoize, gcs_ls
```

Comment:
Import modules not classes/methods inside those modules. Also one import per line